mlir/test/Conversion/ArithToLLVM/arith-to-llvm.mlir

```diff
@@ -478,9 +478,10 @@ func.func @mului_extended_vector1d(%arg0: vector<3xi64>, %arg1: vector<3xi64>) -
 // -----
 
 // CHECK-LABEL: func @cmpf_2dvector(
+//  CHECK-SAME:     %[[OARG0:.*]]: vector<4x3xf32>, %[[OARG1:.*]]: vector<4x3xf32>)
```

Comment:
Daft question - what does `O` stand for in `OARG0`?

Reply:
`[[ARG0]]` was already in use and I wanted to keep the diff as small as possible. So I wrote `[[OARG]]` (for "Original" arg).